modules/openapi-generator/src/main/resources/cpp-pistache-server/model-source.mustache

```diff
@@ -108,6 +124,19 @@ void {{classname}}::fromJson(nlohmann::json& val)
     {{/required}}{{#required}}{{#isString}}{{setter}}(val.at("{{baseName}}"));
     {{/isString}}{{^isString}}{{#isDateTime}}{{setter}}(val.at("{{baseName}}"));
     {{/isDateTime}}{{/isString}}{{/required}}{{/isPrimitiveType}}{{/isListContainer}}{{/vars}}
+    {{#isEnum}}{{#allowableValues}}
```

Comment:
does this not belong in the vars section of the template?

Reply:
When defining an enum as a reusable component schema (as in #1317) of the form

```
components:
  schemas:
    State:
      description: Enum values for states
      enum:
      - idle
      - initialize
      - ready
      - running
      type: string
```
the 'vars' section is not populated at all. This is the case for generators like 'java' and 'go-server' as well, therefore this should be part of the code generation model.